provisioning/transport/http/devdoc/http_requirements.md

```diff
@@ -110,24 +106,22 @@ The `constructor` creates an Http transport object used to communicate with the
 
 **SRS_NODE_PROVISIONING_HTTP_18_022: [** operation status request polling shall be a GET operation sent to 'https://global.azure-devices-provisioning.net/{idScope}/registrations/{registrationId}/operations/{operationId}' **]**
 
-**SRS_NODE_PROVISIONING_HTTP_18_023: [** If the operation status request times out, `register` shall stop polling and call the `callback` with a `TimeoutError` error. **]**
+**SRS_NODE_PROVISIONING_HTTP_18_023: [** If the operation status request times out, `register` shall stop polling and call the `callback` with with the lower level error **]**
 
 **SRS_NODE_PROVISIONING_HTTP_18_024: [** `register` shall deserialize the body of the operation status response into an object. **]**
 
-**SRS_NODE_PROVISIONING_HTTP_18_025: [** If the body of the operation status response fails to deserialize, `register` will throw an `InternalServerError` error. . **]**
+**SRS_NODE_PROVISIONING_HTTP_18_025: [** If the body of the operation status response fails to deserialize, `register` will throw a `SyntaxError` error. **]**
 
 **SRS_NODE_PROVISIONING_HTTP_18_026: [** If the operation status response contains a failure status code, `register` shall stop polling and call the `callback` with an error created using `translateError`. **]**
 
 **SRS_NODE_PROVISIONING_HTTP_18_027: [** If the operation status response contains a success status code with a 'status' of 'Assigned', `register` shall stop polling and call the `callback` with `err` == null and the body containing the deserialized body. **]**
 
-**SRS_NODE_PROVISIONING_HTTP_18_028: [** If the operation status response contains a success status code with a 'status' that is not 'Assigned', `register` shall fire an `operationStatus` event with the deserialized body and continue polling. **]**
+**SRS_NODE_PROVISIONING_HTTP_18_028: [** If the operation status response contains a success status code with a 'status' that is 'Assigning', `register` shall fire an `operationStatus` event with the deserialized body and continue polling. **]**
 
-**SRS_NODE_PROVISIONING_HTTP_18_030: [** If the operation status response has a success code with a 'status' that is not 'Assigned' or  'Assigning', `register` shall call the callback with a `SyntaxError` error and stop polling. **]**
+**SRS_NODE_PROVISIONING_HTTP_18_030: [** If the operation status response has a success code with a 'status' that is any other value, `register` shall call the callback with a `SyntaxError` error and stop polling. **]**
 
 
 ### disconnect(callback: (err) => void);
```

Comment:
>disconnect [](start = 4, length = 10)

missing a few requirements here (depending on state)

Reply:
sorted the requirements to show this better.  Taking this offline for thinking.

---
In reply to: [142297915](https://github.com/Azure/azure-iot-sdk-node/pull/99#discussion_r142297915) [](ancestors = 142297915)